flink-autoscaler/src/main/java/org/apache/flink/autoscaler/utils/DateTimeUtils.java

```diff
@@ -26,7 +26,7 @@
 public class DateTimeUtils {
 
     private static final DateTimeFormatter DEFAULT_FORMATTER =
-            DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+            DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSS");
```

Comment:
Why this change? I think seconds accuracy is sufficient and more easily readable.

Reply:
AFAIK we don't use any milliseconds in the tests, but advance only by full seconds. I think this may have unexpected consequences for the storage and logging layer. If this is a test-only change, then it should go into test.